qbraid/runtime/iqm/_compat.py

```
@@ -0,0 +1,219 @@
+) -> list[str]:
+    """Convert IQM measurement payloads into qiskit-style bitstrings."""
+    formatted_results: dict[int, np.ndarray] = {}
+    shots = requested_shots if expect_exact_shots else 0
+
+    for key, values in measurement_results.items():
+        measurement_key = measurement_key_cls.from_string(key)
+        result_array = np.asarray(values, dtype=int)
+        current_shots = len(result_array)
+
+        if current_shots == 0 and not expect_exact_shots:
+            warnings.warn(
```

Ruff (0.15.7)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)